test/onboard-name-flag.test.ts

```
@@ -0,0 +1,195 @@
+    ...(spec.envOverrides ?? {}),
+  };
+  for (const key of Object.keys(env)) {
+    if (env[key] === undefined) {
+      delete env[key];
+    }
+  }
+
+  const out = spawnSync(process.execPath, [scriptPath], {
+    cwd: repoRoot,
+    encoding: "utf-8",
+    env: env as NodeJS.ProcessEnv,
```

ESLint

[error] 'NodeJS' is not defined.

(no-undef)